.pre-commit-config.yaml

```diff
@@ -8,44 +8,56 @@
 # - Run on all files:   pre-commit run --all-files
 # - Register git hooks: pre-commit install --install-hooks
 #
+ci:
+  autoupdate_interval: monthly
 repos:
   # Autoformat: Python code, syntax patterns are modernized
   - repo: https://github.com/asottile/pyupgrade
     rev: v3.3.1
     hooks:
       - id: pyupgrade
         args:
-          - --py38-plus
+          - --py37-plus
 
   # Autoformat: Python code
-  - repo: https://github.com/psf/black
-    rev: 22.12.0
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v2.0.0
     hooks:
-      - id: black
+      - id: autoflake
+        # args ref: https://github.com/PyCQA/autoflake#advanced-usage
         args:
-          - --target-version=py38
-          - --target-version=py39
-          - --target-version=py310
+          - --in-place
 
   # Autoformat: Python code
   - repo: https://github.com/pycqa/isort
     rev: 5.11.1
     hooks:
       - id: isort
-        args:
-          - --profile=black
```

Comment:
The flags removed here should be set in a pyproject.toml or similar before merge if removed here I think